pyproject.toml

```
@@ -28,15 +28,15 @@ classifiers = [
 ]
 dependencies = [
     "importlib-metadata==7.1.0",
-    "reuse==2.1.0",
+    "reuse==3.0.2",
     "GitPython==3.1.43",
 ]
 
 [project.optional-dependencies]
 tests = [
     "pytest==8.1.1",
     "pytest-cov==5.0.0",
-    "reuse==2.1.0",
+    "reuse==3.0.2",
     "GitPython==3.1.43",
 ]
```

Since they are pinned down as dependencies, there is no need to pin them down here as well.
```suggestion
]
```